src/server/db_slice.cc

```diff
@@ -802,48 +805,53 @@ void DbSlice::FlushSlotsFb(const cluster::SlotSet& slot_ids) {
     return true;
   };
 
+  auto iterate_bucket = [&](PrimeTable::bucket_iterator it) {
+    it.AdvanceIfNotOccupied();
+    while (!it.is_done()) {
+      del_entry_cb(it);
```

Comment:
del_entry_cb is redundant now. why not move its implementation here?

Reply:
you are right, this was just easy for me to test this with moving back and forward with TraverseBuckets/Traverse to see the impact of this change